ansible/configs/ansible-multitier-infra/post_software.yml

```diff
@@ -13,45 +13,75 @@
     - name: Setup lab-user
       include_tasks: ./setup_lab_user.yml
 
+- name: Configure Ansible Tower Environment
+  hosts: bastions
+  become: true
+  tasks:
+    - name: Configure Ansible Tower Environment
+      when: software_to_deploy == "tower"
+      include_tasks: ./configure_tower.yml
```

Comment:
There should not be a need for the `./` in this `include_tasks`

Reply:
Changed